src/pages/Login.js

```diff
@@ -86,7 +86,7 @@ export default function Login(props) {
       }
 
       if (user.userType == 'student') {
-        console.log(userDB);
```

Comment:
I'm guessing we'll fix up the whole student vs. teacher login soon? I know you mentioned this in the demo. Just wasn't sure if you were planning on doing that in this Pull Request or in a new one